### PR TITLE
[TW-91] Doc Add WSDL as supported API format

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ Click __New__, then select __API__ or click __+__.
 
 ![New API](https://assets.postman.com/postman-docs/v8-create-api-modal2.jpg)
 
-Enter a name and a version, then select a schema type and format for your API. You can optionally import an API specification directly at this stage—if you don't, Postman will populate your API with a sample specification you can edit at any time.
+Enter a name and a version, then select a schema type and format for your API. You can optionally import an API specification directly at this stage. if you don't, Postman will populate your API with a sample specification you can edit at any time.
```
Comment (claireriley) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:56]:
```suggestion
Enter a name and a version, then select a schema type and format for your API. You can optionally import an API specification directly at this stage. If you don't, Postman will populate your API with a sample specification you can edit at any time.
```

**Review by claireriley — APPROVED** [review]
Looks good, noted missing capitalization

File: src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ contextual_links:
 
 You can validate your API schema in Postman, and validate your elements (documentation, tests, mock servers, monitors) against a schema. This helps keep your API well-defined and ensure that your elements are in sync with your schema. If there is a validation error on the schema or elements do not match it, Postman will present a list of issues that have been found, as well as fixes for these issues. You can then apply fixes to the API elements and revalidate them.
 
-> Validation is available for OpenAPI 3.0 schemas.
+> Validation is available for OpenAPI 2.0 and 3.0 and WSDL 1.1 and 2.0 schemas.
```
Comment (VShingala) [inline src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md:31]: Validation is not available for OpenAPI 2.0 schema yet.

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 3]
```diff
@@ -53,9 +53,9 @@ Click __New__, then select __API__ or click __+__.
 
 ![New API](https://assets.postman.com/postman-docs/v8-create-api-modal2.jpg)
 
-Enter a name and a version, then select a schema type and format for your API. You can optionally import an API specification directly at this stage—if you don't, Postman will populate your API with a sample specification you can edit at any time.
+Enter a name and a version, then select a schema type and format for your API. You can optionally import an API specification directly at this stage. If you don't, Postman will populate your API with a sample specification you can edit at any time.
 
-> Postman currently supports OpenAPI (versions 1.0, 2.0, and 3.0), RAML (0.8 and 1.0), and GraphQL. Your schema can be defined in JSON, YAML, XML, or GraphQL SDL. Multi-file variants of schemas are currently not supported.
+> Postman currently supports OpenAPI (versions 1.0, 2.0, and 3.0), RAML (0.8 and 1.0), GraphQL, or WSDL (1.1 and 2.0). OpenAPI schemas can be defined in JSON or YAML. RAML schemas must be YAML. GraphQL schemas can be JSON or GraphQL SDL. WSDL schemas must be XML. Multi-file variants of schemas are currently not supported.
```
Comment (VShingala) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:58]: Multi-file variants of schemas are supported via "Import -> Folder" flow for OpenAPI 3.0, RAML (0.8 and 1.0), and WSDL (1.1 and 2.0).

@claireriley It's been a long time but I think we discussed this past year. If I remember correctly.

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 4]
```diff
@@ -122,6 +122,8 @@ Postman directly supports importing the following formats:
```
Comment (VShingala) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:122]: Link for Swagger 2.0 should be https://github.com/postmanlabs/swagger2-postman2/

**Pull request merged**